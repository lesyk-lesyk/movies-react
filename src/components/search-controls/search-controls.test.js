import React from 'react';
import { SearchControls } from './index';
import renderer from 'react-test-renderer';

jest.mock('../button');

// jest.mock('../search-filters');

jest.mock('../search-filters', () => ({
  SearchFilters: 'SearchFilters'
}));

describe('SearchControls', () => {
  it('renders', () => {
    const tree = renderer
      .create(<SearchControls />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
