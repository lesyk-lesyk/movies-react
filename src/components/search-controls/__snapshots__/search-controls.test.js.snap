// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchControls renders 1`] = `
<div
  className="search-controls"
>
  <SearchBy />
  <Button
    className="search-button"
    color="red"
    size="lg"
  >
    Search
  </Button>
</div>
`;
